islandora.module

```diff
@@ -611,6 +612,10 @@ function islandora_permission() {
       'title' => t('Revert datastream history'),
       'description' => t('Revert to a previous version of a datastream.'),
     ),
+    ISLANDORA_ACCESS_INACTIVE_AND_DELETED_OBJECTS => array(
```

Comment:
Good point, @adam-vessey ! Would a warning screen ("you are about to change the state of this object to {inactive|deleted}. You do not have permission to view {inactive|deleted} objects. If you proceed, you will be locked out of this object.") be an acceptable workaround, or do you think the ability to make inactive or deleted should be blocked altogether?